projects/storefrontlib/src/lib/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -1,8 +1,8 @@
-<div class="cx-added-to-cart-dialog" #dialog>
+<div #dialog>
   <!-- Modal Header -->
   <ng-container *ngIf="(loaded$ | async); else loading">
-    <div class="cx-added-to-cart-dialog__header modal-header">
-      <div class="cx-added-to-cart-dialog__title modal-title">
+    <div class="cx-dialog-header modal-header">
```

Comment:
This isn't really a blocker. I've realize I've been somewhat lax on some of the naming of the classes. I don't know if it's really necessary to prefix all the classes with something like `dialog` or `product`. I think it should only be done when it adds value. For instance in this case, the whole styling is for a for a dialog so I question its necessity. What do you think ?

Reply:
Well, this is is the same case as the `cx-button` vs `btn` coming from Bootstrap, if you remove one or the other things brake and then styles are coming from different sources.
All `modal-xxxx` classes come from Bootstrap and then are reused by **ngBootstrap**, so `cx-dialog-header` is our override and sets the ground for plugging "agnostic" properties under that class name... although, i personally like more the name of `cx-modal-header` but then the component has already the word "dialog" in its file name.
I think we should keep the override but i'm open to suggestions on the naming.

Reply:
I understand what you are saying. What I was wondering is if it would make it easier for you guys or for anyone extending the style to have `cx-header` instead of `cx-dialog-header`?

Reply:
I would say semantically `cx-dialog-header` is more accurate, even if longer, than just `cx-header` which could lead to think this is somehow related to the `<header>` element.